dts/bindings/sensor/st,lis2dh-i2c.yaml

```diff
@@ -2,7 +2,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 description: |
-    STMicroelectronics LIS2DH 3-axis accelerometer accessed through I2C bus
+    STMicroelectronics LIS2DH 3-axis accelerometer accessed through I2C bus.
+
+    This binding is supported by CONFIG_LIS2DH.
```

Comment:
FYI, the reason I didn't take this approach is because the bindings are at least notionally OS agnostic.

As a practical matter, it seems unlikely to me that anybody besides Zephyr will ever be using this bindings language, but I don't know. @b0661 ?